tests/unit/test___init__raises.py

```diff
@@ -31,6 +31,15 @@ def function_1():
         ),
         pytest.param(
             '''
+def _function_1():
+    """Docstring 1."""
+    raise Exc1
+''',
+            (),
+            id="private function raises single exc docstring no raises section",
```

Comment:
Could we also have a private function that has a raises section but the function does not raise?

Reply:
and then maybe a private function that does raise and it has the raises section in the docstring